.flake8

```diff
@@ -0,0 +1,18 @@
+[flake8]
+max-line-length = 120
+max-complexity = 10
+ignore =
+    E501,  # line too long, it is covered by pylint
+    E722,  # bare except, bad practice, to be removed in the future
+    F401,  # imported but unused, too many violations, to be removed in the future
+    F811,  # redefinition of unused, to be removed in the future
+    C901,  # code flow is too complex, too many violations, to be removed in the future
+    W504   # line break after binary operator effect on readability is subjective
+exclude = 
+    tests
```

Comment:
should we target to get this covered as well?
not now but may be later

Reply:
Thats the plan!